scripts/configs/behavior_pick_place_evaluation.yaml

```diff
@@ -0,0 +1,75 @@
+# A configuration file for evaluation on 5 pick and place BEHAVIOR tasks.
+# python scripts/supercloud/launch.py \
+# --user wmcclinton \
+# --config behavior_pick_place_data_evaluation.yaml
+---
+APPROACHES:
+  pnad_search: # PNAD search approach.
+    NAME: "nsrt_learning"
+    FLAGS:
+      strips_learner: pnad_search
+  cluster_and_intersect:  # Cluster-and-intersect (RLDM) baseline.
+    NAME: "nsrt_learning"
+    FLAGS:
+      strips_learner: "cluster_and_intersect"
+      disable_harmlessness_check: True
+  cluster_and_search:  # LOFT baseline.
+    NAME: "nsrt_learning"
+    FLAGS:
+      strips_learner: "cluster_and_search"
+      disable_harmlessness_check: True
+  pred_error:  # Prediction error baseline that optimizes via hill climbing.
+    NAME: "nsrt_learning"
+    FLAGS: 
+      strips_learner: "cluster_and_intersect_sideline_prederror" 
+      disable_harmlessness_check: True
+  gnn_shooting:  # Model-based GNN option policy baseline.
+    NAME: "gnn_option_policy"
+  gnn_modelfree:  # Model-free GNN option policy baseline.
+    NAME: "gnn_option_policy"
+    FLAGS:
+      gnn_option_policy_solve_with_shooting: False
```

Comment:
add a note about the fact that this should be commented out until `gnn_shooting` is run first